db/migrate/20191114141607_add_current_school_to_maths_and_physics_eligibilities.rb

```diff
@@ -0,0 +1,5 @@
+class AddCurrentSchoolToMathsAndPhysicsEligibilities < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :maths_and_physics_eligibilities, :current_school, type: :uuid, foreign_key: {to_table: :schools}
```

Comment:
Can we change the commit name to something like `Add current_school to Maths and Physics eligibility`?